src/stores/contactmoment/index.ts

```diff
@@ -1,102 +1,151 @@
 import type { Klant } from "@/features/klant/types";
 import type { Zaak } from "@/features/zaaksysteem/types";
+import { getFormattedUtcDate } from "@/services";
 import { defineStore } from "pinia";
 import { resetAllState } from "../create-store";
-import type { NieuweKlant } from "./types";
 export * from "./types";
 
-export type ContactmomentZaak = Zaak & { shouldStore: boolean };
+export type ContactmomentZaak = { zaak: Zaak; shouldStore: boolean };
+export type ContactmomentContactVerzoek = {
+  url: string;
+  attendee: string;
+  isDirty: boolean;
+  isSubmitted: boolean;
+};
 
-interface ContactmomentState {
-  contactmomentLoopt: boolean;
+export interface Vraag {
   zaken: ContactmomentZaak[];
-  klanten: { klant: Klant; shouldStore: boolean }[];
   notitie: string;
-  contactverzoek: { url: string; medewerker: string } | undefined;
-  nieuweKlant: NieuweKlant | undefined;
+  contactverzoek: ContactmomentContactVerzoek;
```

Comment:
Probably OOS for this PR, but couldn't there be multiple contactverzoeken in one vraag?

Reply:
Interesting. There might, we would have to ask. Before introducing the concept of a question, there was only one contactmoment, to which you could only add one contactverzoek. If the users want to be able to add multiple contactverzoeken, I think it warrants it's own story/issue.